bot/src/discord_commands/pronounce.js

```diff
@@ -66,33 +65,73 @@ function createLHString(pronounceInfo) {
   return str;
 }
 
+function getPitchPattern(pronounceInfo) {
+  const alternatives = {
+    heio: '平板 / 尾高 \nHeiban / Odaka',
+    naka: '中高 \nNakadaka',
+    atama: '頭高 \nAtamadaka',
+  };
+  const pitch = pronounceInfo.pitchAccent; // 食べる -> たべる -> Low High Low -> [False, True, False]
+  if (!pitch[0]) {
+    delete alternatives.atama;
+    Object.entries(pitch).forEach(([index, element]) => {
+      const i = parseInt(index, 10);
+
+      if (i > 0 && !element) {
+        delete alternatives.heio;
+      } else if (i === pitch.length && !element) {
+        delete alternatives.naka;
+      }
+    });
+  } else {
+    delete alternatives.heio;
+    delete alternatives.o;
```

Comment:
It doesn't look to me like `o` is ever set as a property on `alternatives`. Is it supposed to be deleting something else here? Or maybe this line is unnecessary?